cicd/3-app/deploy-development-stack.rb

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'fileutils'
+require 'open3'
+require 'erb'
+require 'json'
+
+# Default options
+options = {
+  profile: 'codeorg-dev',
+  region: 'us-east-1',
+  stack_name: 'javabuilder-dev',
```

Comment:
We need to support multiple development Javabuilder Stacks provisioned in the same AWS Account & Region. I recommend we set `stack_name` and `subdomain` to `javabuilder-[feature branch]`. We could grab the current branch name and maybe take the text after the last `/` in the branch name. At the very least let's make `stack_name` and `subdomain_name` required command line options.

Reply:
The naming convention already established for development environments is `javabuilder-dev-<branch-name>`  per https://github.com/code-dot-org/javabuilder/blob/main/cicd/2-cicd/cicd.template.yml

When [provisioning a full CICD pipeline](https://github.com/code-dot-org/javabuilder/blob/main/cicd/2-cicd/cicd.template.yml), `javabuilder-<branchname>` is used, appending `-test` for [the pipeline's Test environment](https://github.com/code-dot-org/javabuilder/blob/main/cicd/2-cicd/cicd.template.yml).